projects/centre_console/inc/regen_braking_toggle.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+// A module to toggle the regen braking state, and reverts the state if the
+// ACK on the REGEN_BRAKING message fails
+// Requires the event queue, interrupts, soft timers, gpio, and CAN to be initialized.
+
+#include <stdbool.h>
+
+#include "can.h"
+#include "can_ack.h"
+#include "can_msg_defs.h"
+#include "can_pack.h"
+#include "can_transmit.h"
+#include "event_queue.h"
+#include "status.h"
+
+StatusCode regen_braking_init();
+
+bool get_regen_braking_state();

```

Comment:
Nitpicks: the parameter lists here should be `(void)` rather than `()`, same in the .c file too.